components/rmrk/Gallery/GalleryCard.vue

```diff
@@ -19,11 +19,7 @@
           :alt="title"
           customClass="gallery__image-wrapper" />
 
-        <PreviewMediaResolver
-          v-else
-          :src="animatedUrl"
-          :metadata="metadata"
-          :mimeType="type" />
+        <PreviewMediaResolver v-else :src="animatedUrl" :metadata="metadata" />
```

Comment:
you forgot to pass `:mimeType="type"`

Reply:
hmm, maybe I do not really understand this line correctly

may I know where the `type` variable coming from? when I open this [link](http://localhost:9090/rmrk/collection/7cf9daa38281a57331-BSS?page=1) in local there is a console error `Property or method "type" is not defined on the instance but referenced during render.`

should the parent components bring `type` as a props?

Reply:
it seems fine if there are no `:mimeType` props, because `PreviewMediaResolver` component will get `mimeType` on `onMounted` and use `idb-keyval` also. if we fetch `mimetype` in the parent component, it will re-fetch every time

wyt?